understanding/focus-not-obscured-minimum.html

```diff
@@ -380,90 +275,62 @@ <h4>User openable, persistent disclosures</h4>
                </section>
                <section id="dialogs">
                   
-                  <h3>Modal dialogs</h3>
+                  <h3>モーダルダイアログ</h3>
                   
                   
-                  <p>A properly constructed modal dialog will always pass this SC. Even if it appears directly
-                     on top of an item with focus, the dialog takes focus on appearance, and thus the item
-                     receiving focus -- the dialog or one of its components -- is visible. A properly constructed
-                     modal maintains that focus and prevents interaction outside the modal until it is
-                     dismissed.
+                  <p>適切に構成されたモーダルダイアログは、常にこの達成基準に合格する。たとえフォーカスを持つ項目の上に直接表示されたとしても、ダイアログは外観上フォーカスを受け取り、したがってフォーカスを受け取る項目 (ダイアログ又はそのコンポーネントの一つ) が表示される。適切に構成されたモーダルは、そのフォーカスを維持し、モーダルが閉じられるまでモーダル外のインタラクションを防ぐ。
                   </p>
                   
-                  <p>A dialog-like overlay that does not take focus on appearance and does not either constrain
-                     interaction to the overlay or dismiss itself on loss of focus (thus allowing focus
-                     to exit into the content behind it) will be at risk of failing this SC, where it is
-                     positioned such that it can obscure other focusable items.
+                  <p>外観上でフォーカスを受け取らず、かつオーバーレイに対してインタラクションを制限しない、又はフォーカスを失ったときに自身を閉じない (したがって、フォーカスが背後のコンテンツに抜けるようにする) ダイアログのようなオーバーレイは、他のフォーカス可能な項目を隠すことができるように配置されており、この達成基準で失敗する危険性がある。
```

Comment:
"on appearance" など細かいところを調整してこんなかんじでどうでしょう?
「かつ」と「又は」が両方あってわかりにくいですが、原文にも and と or があるので……

```suggestion
                  <p>ダイアログのようなオーバーレイが、表示された際にフォーカスを受け取らず、かつオーバーレイに対してインタラクションを制限しない、又はフォーカスを失ったときに自身を閉じない (したがって、フォーカスが背後のコンテンツに抜けるようになる) 場合に、他のフォーカス可能な項目を隠すことができるように配置されていると、この達成基準で失敗する危険性がある。
```